### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][NVDA][MAS4.2.1][IntelliJ][Wipro]::NVDA is announcing tab index inappropriately in Run/Debug configurations window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1502]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Navigate to and select "edit configurations" present at right side of the dialog.
4. "Run/Debug Configurations" dialog will open.
5. In "Run/Debug Configurations", Navigate to "Azure Web App" expand/collpase button present in left side then observe the NVDA Announcement

**Expected Result:**
**NVDA Behavior:**
NVDA should announce
level 1  Azure Web App  expanded  selected  1 of 2
level 2  Azure Web App: sample: sample  1 of 1
level 1  Defaults  collapsed  2 of 2

**Actual Result:**
**NVDA Behavior:** NVDA is announcing as 
level 2  Azure Web App  expanded  selected  1 of 2
level 3  Azure Web App: sample: sample  selected  1 of 1
level 2  Defaults  collapsed  selected  2 of 2

**Jaws Behavior:** Jaws is announcing as expected
Azure Web Appexpanded
1 item
Azure Docker Supportcollapsed
Applicationcollapsed
Azure Docker Supportcollapsed
level 2
Web App for Containers: untitled4:untitled4
1 of 1
level 1
Azure Web Appexpanded
1 item
level 2
Azure Web App: untitled4:untitled4
1 of 1
level 1
Defaultscollapsed
4 of 4

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![edit configuarions-position](https://user-images.githubusercontent.com/22956130/36141936-995c6de4-10cc-11e8-88b3-00f8f11ad985.jpg)
![tree items position is incorrect in run or debug configurations window](https://user-images.githubusercontent.com/22956130/36141948-a363148c-10cc-11e8-89c8-90749c5c2df7.jpg)

**Comment by Eskibear** [comment]
Content hierarchy is provided by IntelliJ run/debug configuration, out of the toolkit's control.

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by Eskibear** [comment]
And it's out of the toolkit's scope.

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749940
